lib/auth/utils.ts

```
@@ -0,0 +1,29 @@
+  const encoder = new TextEncoder();
+  const data = encoder.encode(value);
+  const hashBuffer = await window.crypto.subtle.digest('SHA-256', data);
+  // Uint8Array를 hex 문자열로 변환
+  const hashArray = Array.from(new Uint8Array(hashBuffer));
+  const hashHex = hashArray.map(b => b.toString(16).padStart(2, '0')).join('');
+  return hashHex;
+}
+
+/** 랜덤 Nonce 생성 */
+function generateRandomNonce(length: number = 32): string {
+  return generateRandomString(length);
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)